frontend/src/components/Jobs.vue

```diff
@@ -6,16 +6,45 @@
   </template>
   <div
     v-if="!loading"
-    id="jobs"
   >
-    <template
-      v-for="job in jobs"
-      :key="job"
+    <div
+      class="jobs"
     >
-      <Job :job="job" />
-    </template>
+      <template
+        v-for="job in visibleJobs"
+        :key="job"
+      >
+        <Job
+          :job="job"
+          :hidden="false"
+        />
+      </template>
+    </div>
+    <div v-if="hiddenJobs.length !==0">
+      <hr>
+      <div class="hidden_jobs">
+        {{ hiddenJobs.length }} hidden jobs
```

Comment:
I am not sure should we call it **hidden jobs** or not

Reply:
@sumanmaity1234 I think this is the only open point now, what do you think?

Reply:
Hi @sweiler , I will think little bit more about it also talk with couple of other contributor, if everyone thinks fine then I'll mark this comment as resolve